src/ui/prompts/listPrompts/listOptionsPrompt.ts

```diff
@@ -118,7 +118,9 @@ export abstract class ListOptionsPrompt<T> {
         try {
             switch (response) {
                 case 'add':
-                    newList.push(await this.promptAddItem())
+                    const newItem = await this.promptAddItem()
+                    newItem.value.id = newList.length
```

Comment:
should this not be `newList.length - 1`? Or am I incorrectly assuming that the id field corresponds to the index?

Reply:
yeah the id corresponds to the index but the item is being added to the list _after_ it's id is set so `newList.length` makes sense. First item gets 0, second item gets 1, third item gets 2... so on

Reply:
ah true!